CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-31
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```

Comment:
we should carry over a few of the important announcements from previous release. for example Windows server retiring from K8s 1.34 one

Reply:
Copied that one over. Should I carryover all the others or something else specific?